### ISSUE TITLE
export * as ns from ... gives error

### ISSUE THREAD

**Issue by hacker-DOM** [babel/babel-eslint#813]
Hi!

I am using `@babel/plugin-proposal-export-namespace-from` for code

```js
export * as T from './types'
```

Despite having a correct set up of Babel & Eslint, I get two linting errors. I'm disabling those rules but just thought I'd let you know.

I am on `babel-eslint@11.0.0-beta.0`...

**Comment by hacker-DOM** [comment]
Update: Doesn't work on `-beta.2` either

**Comment by eankeen** [comment]
i get this as well, receiving a
```txt
Parsing error: Unexpected token as
```

with the abridged configuration looking like this

```js
{
  "parserOptions": {
    "parser": "babel-eslint",
    "ecmaVersion": 2020,
    "sourceType": "module",
    "ecmaFeatures": {
      "impliedStrict": true,
      "modules": true
    },
},
 "env": {
    "node": true,
    "es6": true,
    "es2017": true,
    "es2020": true,
    "jest": true,
    "jest/globals": true
  },
  }
```

not sure why this may be the case?

**Comment by JLHwung** [comment]
Fixed in `@babel/eslint-parser` 7.11.0, see https://babel.dev/blog/2020/07/13/the-state-of-babel-eslint#the-future for the renaming guide.
